lib/croods/controller/collection.rb

```diff
@@ -5,10 +5,28 @@ class Controller < ActionController::API
     module Collection
       protected
 
+      def page_param
+        params[resource.page_attribute_name]
+      end
+
+      def paginate_collection(list)
+        paginate(list.send(resource.page_method_name, page_param))
+      end
+
+      def sort(list)
+        if list.respond_to? resource.sort_by
+          list.public_send(resource.sort_by, params[:order_by], params[:order])
```

Comment:
I think it's enough to have only 1 pair of parameters, for now. This can be extended in the future to sort by multiple fields.

Reply:
I agree.